examples/resources/opslevel_integration_google_cloud/resource.tf

```diff
@@ -0,0 +1,11 @@
+resource "opslevel_integration_google_cloud" "dev" {
+  client_email            = "hello-world-tf@powerful-surf-427415-v1.iam.gserviceaccount.com"
+  name                    = "GCP Dev Integration"
+  ownership_tag_keys      = toset(["opslevel_team", "team", "owner"])
```

Comment:
why `toset` ?

Reply:
@rocktavious just double checked and it is unnecessary, I will remove this from the examples and test code